core/examples/minimal_example.py

```diff
@@ -19,15 +19,80 @@
 |Content Cell  | `Content Cell` | Content      |
 |Content Cell  | Content Cell   | Content      |
 
-
 ---
 
     #!python
     def hello():
         print('Hello World')
+---
+If you want to use math formulas, you have several different options. You can
+use double dollar signs:
+
+```
+$$ \left(\frac{\sqrt x}{y^3}\right) $$
+```
+
+Result: $$ \left(\frac{\sqrt x}{y^3}\right) $$
+
+Or you can wrap it between special commands like this:
+
+```
+\begin{equation}
+\cos (2\theta) = \cos^2 \theta - \sin^2 \theta \label{my_cos_equation}.
+\end{equation}
+```
+
+All equations with labels can easily be referred to in the text as
+```\eqref{my_cos_equation}```. If you want an equation without numbering
+add "notag":
```

Comment:
...and then this paragraph can be changed to:

All equations with labels can easily be referred to in the text as
```\eqref{my_cos_equation}```, resulting in \eqref{my_cos_equation}. 

If you want an equation without numbering add `\notag`: